lib/core-server/src/build-dev.ts

```diff
@@ -110,9 +110,14 @@ export async function buildDevStandalone(options: CLIOptions & LoadOptions & Bui
     return;
   }
 
+  // Get package name and capitalize it e.g. @storybook/react -> React
+  const packageName = name.split('@storybook/')[1] || '';
```

Comment:
Make sure the length > 0

Reply:
The `|| ''` will do the same, serving as a failsafe mechanism in case the length is not `> 0`.

I can change the code to:

```js
const packageName = name.split('@storybook/').length > 0 ? name.split('@storybook/')[1] : '';
```

if you prefer! In the end, they both do the same

Reply:
TIL `['hello'][1]` returns `undefined`!

What about:

```
const packageName = name.split('@storybook/').length > 0 ? name.split('@storybook/')[1] : name;
```

Reply:
Done! The unofficial framework packages will also be shown with a capital letter:

`Storybook 6.4 for Unofficial-framework-name`